### ISSUE TITLE
implement "web-share" on mobile devices instead of copying the event-type

### ISSUE THREAD

**Issue by PeerRich** [calcom/cal.com#1768]
this button:

<img width="548" alt="image" src="https://user-images.githubusercontent.com/8019099/153261471-66ac9831-60d0-421e-a1e0-89e3a8ba7f4a.png">

should open the android or iOS "web share" bottom panel:

<img width="303" alt="image" src="https://user-images.githubusercontent.com/8019099/153261553-486d0b4b-eba8-4112-b350-cbc43b39420a.png">

we can use: https://web.dev/web-share/

```
if (navigator.share) {
  navigator.share({
    title: 'web.dev',
    text: 'Check out web.dev.',
    url: 'https://web.dev/',
  })
    .then(() => console.log('Successful share'))
    .catch((error) => console.log('Error sharing', error));
}
```

**Comment by Jenietoc** [comment]
Hi @PeerRich I want to work on this issue can you assign me?

**Comment by PeerRich** [comment]
> Hi @PeerRich I want to work on this issue can you assign me?

assigned! make sure it only opens in web-share on mobile and falls back to copying if web-share is not supported.

**Comment by PeerRich** [comment]
assigned you @Jenietoc

**Comment by Jenietoc** [comment]
Hey @PeerRich  how are you? do you think this is a good solution? (I know I have to internalize the strings used).
![code](https://user-images.githubusercontent.com/89233604/153517370-310f7947-66c0-48fd-be58-ccefe601b710.png)

**Comment by PeerRich** [comment]
> Hey @PeerRich  how are you? do you think this is a good solution? (I know I have to internalize the strings used).
> 
> ![code](https://user-images.githubusercontent.com/89233604/153517370-310f7947-66c0-48fd-be58-ccefe601b710.png)
> 
> 

yes looks good! only missing is wrapping hard coded strings in {t("key")}
